web.py

```diff
@@ -229,7 +229,14 @@ def get_scan_coords():
     markers = []
     if config.BOUNDARIES:
         from shapely.geometry import mapping
-        coords = mapping(config.BOUNDARIES)['coordinates'][0]
+        mapping = mapping(config.BOUNDARIES)['coordinates']
+        coords = mapping[0]
+        if len(mapping) > 1:
+            for i in range(1, len(mapping)):
+                markers.append({
+                        'type': 'scanblacklist',
+                        'coords': mapping[i]
+                    })
```

Comment:
how about:
```python
for blacklist in mapping[1:]:
    markers.append({
        'type': 'scanblacklist',
        'coords': blacklist
    })
```
Should make the `if` statement unnecessary while being more Pythonic.